cloud/services/compute/network.go

```diff
@@ -103,6 +104,26 @@ func (s *Service) DeleteNetwork() error {
 		}
 	}
 
+	// Delete routes associated with network
+	filterString := fmt.Sprintf("description=k8s-node-route name=%s-*", s.scope.Name())
```

Comment:
how about other routes, like the default routes, are they deleted elsewhere?

Reply:
They are auto-deleted when network is deleted